components/Calendar/index.js

```diff
@@ -1,20 +1,79 @@
 import Calendar from "react-calendar";
 import "react-calendar/dist/Calendar.css";
 import { useState } from "react";
-// implement styles.js later
+import useCalorieStore from "../../utils/useCalorieStore";
+import styled from "styled-components";
 
 export default function HomeCalendar() {
   const [date, setDate] = useState(new Date());
+  const { history } = useCalorieStore();
+
+  function getTileClassName(date) {
+    if (
+      history.find(
+        (entry) => entry.date === date.getTime() && entry.entry.wasExceeded
+      )
+    ) {
+      return "react-calendar__tile--wasExceeded";
+    } else if (
+      history.find(
+        (entry) => entry.date === date.getTime() && !entry.entry.wasExceeded
+      )
+    ) {
+      return "react-calendar__tile--wasNotExceeded";
+    }
+  }
 
   return (
-    <>
+    <CalendarWrapper>
       <Calendar
+        value={date}
+        selectRange={false}
+        maxDate={new Date(2025, 4, 4)} // replace with today + X later
+        minDate={new Date(2023, 3, 1)} // replace with earliest stored date later
+        minDetail="month"
+        formatShortWeekday={formatShortWeekday}
+        formatMonthYear={formatMonthYear}
+        // ---
         onChange={(date) => {
           setDate(date);
           console.log(date);
         }}
-        value={date}
+        // ---
+        className="calendar-style"
+        tileClassName={({ date }) => getTileClassName(date)}
       />
-    </>
+    </CalendarWrapper>
   );
 }
+
+// calendar styling:
+
+const formatShortWeekday = (locale, date) => {
+  const weekdays = ["Su", "Mo", "Tu", "We", "Th", "Fr", "Sa"];
+  return weekdays[date.getDay()];
+};
+
+const formatMonthYear = (locale, date) => {
+  return new Intl.DateTimeFormat(locale, {
+    year: "2-digit",
+    month: "long",
+  }).format(date);
+};
+
+const CalendarWrapper = styled.div`
+  .react-calendar__tile--wasExceeded {
+    background-color: crimson;
+  }
+  .react-calendar__tile--wasNotExceeded {
+    background-color: #32de84;
+  }
```

Comment:
I think of something like this:
```suggestion
    background-color: ${({ tileClassName }) =>
    tileClassName === "wasExceeded ? "crimson" : "#32de84"};
  }
```

Note that you need to adapt the return value of the `getTileClassName` function to return either `wasExceeded` or `wasNotExceeded` (or simply `true` and `false`, but then you should rename the function and the prop to reflect that the return value is a boolean.